_harp/apply/london.ejs

```diff
@@ -0,0 +1,17 @@
+<div class="section-copy">
+	<div class="container apply-page">
+		<h1 align='center'>LONDON</h1>
+
+		<p align='center'>Apply today for our next course: 23 October, 2017 - February 9, 2018</p>
+
+		<div align='center'>
+		  <a href="https://goo.gl/forms/CERu8qvH6kjL7pZG3" target="_blank"><button class="button-three">APPLY</button></a>
+		</div>
+
+		<%- partial("_london.md") %>
+
+		<div align='center'>
+		  <a href="https://goo.gl/forms/CERu8qvH6kjL7pZG3" target="_blank"><button class="button-three">APPLY</button></a>
```

Comment:
so I'm not sure about this second apply, repeatitive for my taste, @jsms90 , @emilyb7 what do you think?